recipe_Menager.py

```diff
@@ -2,26 +2,105 @@ class RecipeManager:
     def __init__(self):
         self.recipes = []
 
-    @property
     def display_recipes(self):
         for recipe in self.recipes:
-            print(recipe)
-            return self.recipes
-            
-    @display_recipes.getter
-    def display_recipes(self):
-      return str(self.recipes)
+            print(recipe._title) #title instead the name 
+            return 
+        if not self.recipes:
+            print("No recipes to display.")
+            return
+
+
+    def add_recipe(self, recipe_to_add: Recipe):
+        self.recipes.append(recipe_to_add)
+    def add_recipe_from_input(self):
+        title = input("Enter the title of the dish: ")
+        description = input("Enter the description: ")
+        no_servings = int(input("Enter the number of servings: "))
+        calories_per_portion = int(input("Enter the calories per portion: "))
+        ingredients = input("Enter the ingredients: ").split(",")
+        instructions = input("Enter the instructions: ").split(",")
+
+        new_recipe = Recipe(title)
+        new_recipe._description = description  # Set description directly
```

Comment:
The setting of attributes in the recipe should be done via decorated access methods.